vtrace/platforms/posix.py

```diff
@@ -61,6 +61,14 @@ def handleAttach(self):
         # break after our library load events to make things easy
         self.runAgain(False)  # Clear this, if they want BREAK to run, it will
         self.fireNotifiers(vtrace.NOTIFY_BREAK)
+        # POSIX hack - Windows signals on library load, POSIX doesn't
+        self._LibraryLoadHack()
+        
+    def _LibraryLoadHack(self):
```

Comment:
Maybe call it `LibraryLoadHook`?

Reply:
can do.  but it literally is a hack because PTRACE doesn't notify on library loads.

Reply:
done.